projects/1.20/assets/eternal-tales/eternal_tales/lang/zh_cn.json

```
@@ -123,12 +123,12 @@
 	"advancements.get_forest_ranger_armor.title": "护林员",
 	"advancements.get_gardeners_pot.descr": "获得园丁的盆栽",
 	"advancements.get_gardeners_pot.title": "年轻的园丁",
-	"advancements.get_glowberry_cookie.descr": "Get Glowberry Cookie",
-	"advancements.get_glowberry_cookie.title": "I'm glowing!",
+	"advancements.get_glowberry_cookie.descr": "制作发光蓝莓曲奇",
```

```suggestion
	"advancements.get_glowberry_cookie.descr": "获得发光浆果曲奇",
```
查看源代码可知，此处判定是物品栏内是否有对应物品，故使用`获得`更恰当。
另外`glowberry`是[发光浆果](https://zh.minecraft.wiki/w/%E5%8F%91%E5%85%89%E6%B5%86%E6%9E%9C)。